screens/UserListScreen.tsx

```diff
@@ -1,12 +1,27 @@
+import { NativeStackScreenProps } from "@react-navigation/native-stack";
 import { useCallback, useEffect, useState } from "react";
-import { FlatList, Image, StyleSheet, Text, View } from "react-native";
+import {
+  FlatList,
+  Image,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from "react-native";
+import { RootStackParamList } from "../App";
+
+type Props = NativeStackScreenProps<RootStackParamList, "UserListScreen">;
+
+export type UserData = {
+  name: string;
+  thumbnail: string;
```

Comment:
thumbnailはデータが入ってるわけでなくURLだとおもうので、こっちが親切かも
```suggestion
  thumbnailUrl: string;
```